nd2reader/raw_metadata.py

```diff
@@ -139,11 +139,15 @@ def _process_channels_metadata(self, metadata):
 
     def _get_channel_validity_list(self, metadata):
         try:
-            validity = self.image_metadata[six.b('SLxExperiment')][six.b('ppNextLevelEx')][six.b('')][0][
-                six.b('ppNextLevelEx')][six.b('')][0][six.b('pItemValid')]
+            validity = self.image_metadata[six.b('SLxExperiment')][six.b('ppNextLevelEx')][six.b('')][
```

Comment:
I am not sure if it makes sense to refactor it. Please advise me if I need to try to remove this duplication. For readability, it may make sense to keep the current state.